soc/riscv32/litex-vexriscv/Kconfig.defconfig

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2018 - 2019 Antmicro <www.antmicro.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_RISCV32_LITEX_VEXRISCV
+
+config SOC
+	string
+	default "litex-vexriscv"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	int
+	default 100000000
+
+config RISCV_HAS_CPU_IDLE
+	bool
+
+config RISCV_HAS_PLIC
+	bool
+
+config NUM_IRQS
+	int
+	default 12
+
+# currently n-selecting, because of UART Tx interrupt related bug
+config SERIAL_SUPPORT_INTERRUPT
```

Comment:
I assume from the comment you are saying that interrupt's aren't supported.  Is this really accomplishing what you want?

Reply:
You are right, that was done wrong. I removed this line from soc-level Kconfig and added `depends on !SERIAL_SUPPORT_INTERRUPT` in uart-level Kconfig.